prog/dftb+/lib_dftbplus/initprogram.F90

```diff
@@ -4868,11 +4893,14 @@ subroutine TReksCalc_init(reks, reksInp, electronicSolver, orb, spinW, nEl,&
       case (electronicSolverTypes%onlyTransport)
         call error("REKS is not compatible with OnlyTransport-solver")
       case(electronicSolverTypes%qr, electronicSolverTypes%divideandconquer,&
-          & electronicSolverTypes%relativelyrobust, electronicSolverTypes%elpa)
+          & electronicSolverTypes%relativelyrobust)
         call REKS_init(reks, reksInp, orb, spinW, nSpin, nEl(1), nExtChrg, extChrg,&
             & blurWidths, is3rd, isRangeSep, tForces, tPeriodic, tStress, tDipole)
-      case(electronicSolverTypes%omm, electronicSolverTypes%pexsi, electronicSolverTypes%ntpoly)
-        call error("REKS is not compatible with density matrix ELSI-solvers")
+      case(electronicSolverTypes%magma_gvd)
```

Comment:
The magma solver should behave like a standard serial eigensolver. Is there a particular reason for this incompatibility?

Reply:
@bhourahine Currently, it is not tested so I do not include the case. If needed, I will test and add the magma solver case for REKS calculation if there is no problems.

Reply:
Also, there may exist a problem with MAGMA sovler. Currently, REKS code is openmp-parallelized, especially in gradient part. So, I am not sure about the process how the OMP parallelization works under GPU case.

Reply:
Probably would not cause problems, but this can be revisited after the 20.1 release.